lib/mixins/dropdown.js

```diff
@@ -94,9 +96,18 @@ export default {
                         el.removeEventListener('mouseover', this.noop);
                     });
                 }
+                document.removeEventListener('keydown', this.keydownListener);
```

Comment:
I'm not sure as trigger is only possible with browser and didn't made any special SSR error during dev.